### PR TITLE
486623: Add decommission endpoint for unneeded services

### PR DESCRIPTION
**Pull request by philsegal**

### CODE REVIEW

File: src/api/decommission-service/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { decommissionServiceController } from '~/src/api/decommission-service/controller.js'
+
+const decommissionService = {
+  plugin: {
+    name: 'decommission-service',
+    register: async (server) => {
+      await server.route([
+        {
+          method: 'DELETE',
```
Comment (flurdy) [inline src/api/decommission-service/index.js:9]: Do we do proper REST methods? 
Or do we need a fallback  POST? 
I see there is one PATCH in SSOP.

Comment (flurdy) [reply src/api/decommission-service/index.js:9]: Since this is a backend I'd guess it is fine as not browser based

**Review by flurdy — APPROVED** [review]

File: src/api/decommission-service/index.js (hunk 1 above)
Comment (philsegal) [reply src/api/decommission-service/index.js:9]: Seems fairly easy to change down the line if we need to 🤷‍♂️

File: src/api/decommission-service/controller.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        serviceName: Joi.string().required()
+      }),
+      failAction: () => Boom.boomify(Boom.badRequest())
+    }
+  },
+  handler: async (request, h) => {
+    const serviceName = request.params.serviceName
+
+    await removeStatus(request.db, serviceName)
+    await portalBackEndDecommissionService(serviceName)
```
Comment (feedmypixel) [inline src/api/decommission-service/controller.js:26]: Are this transactions?

Comment (philsegal) [reply src/api/decommission-service/controller.js:26]: no.

File: src/api/decommission-service/index.js (hunk 1 above)
Comment (feedmypixel) [reply src/api/decommission-service/index.js:9]: > Do we do proper REST methods?
FYI USBE uses most of the REST methods

**Pull request merged**